corehq/util/celery_utils.py

```diff
@@ -39,9 +39,8 @@ class TaskInfo(object):
     def __init__(self, _id, name, time_start=None):
         self.id = _id
         self.name = name
-        # http://stackoverflow.com/questions/20091505/celery-task-with-a-time-start-attribute-in-1970
         if time_start:
-            self.time_start = datetime.fromtimestamp(time() - kombu.five.monotonic() + time_start)
+            self.time_start = datetime.fromtimestamp(time_start)
```

Comment:
@calellowitz  @millerdev Just want to be sure that my understanding of it is correct.
`time_start` now no longer is montonic in celery. See the comment http://stackoverflow.com/questions/20091505/celery-task-with-a-time-start-attribute-in-1970

Reply:
I don't think I have enough context to answer that or whether it matters. We should probably test this to ensure it does the right thing with the new version of Celery.